src/udiskslinuxprovider.c

```diff
@@ -355,13 +366,10 @@ on_etc_udisks2_dir_monitor_changed (GFileMonitor     *monitor,
       event_type == G_FILE_MONITOR_EVENT_CHANGES_DONE_HINT)
     {
       gchar *filename = g_file_get_basename (file);
+      gchar *id = dup_id_from_config_name (filename);
       if (g_str_has_suffix (filename, ".conf"))
```

Comment:
Hi!  Probably it's not too important, but the code looks weird.  This didn't quite come out as I was hoping.  If you look in the diff and go a few lines further up, you can see the definition of the function `dup_id_from_config_name()`, which is newly added.

`dup_id_from_config_name()` copies `filename` and shortens it by 5 bytes.  The function assumes that `filename` ends in `.conf`.  But, this code only checks that `filename` ends in `.conf` _after_ it's already called `dup_id_from_config_name()`.